assets/css/style.css

```diff
@@ -639,3 +639,25 @@ header {
 .paragraph {
   font-size: 21px;
 }
+
+/* Styling the form */
+.form {
+  display: flex;
+  justify-content: space-between;
+}
+
+input {
+  font-family: Cambria, Cochin, Georgia, Times, 'Times New Roman', serif;
+  border-radius: 0.5em;
+}
+
+label {
```

Comment:
Here happens the same again, if you do something like this in the middle of the code, it can take some time to fix it later. If you want to target all elements of the type, put it before all classes help you to find it later.